### DISCUSSION TITLE
Multi valued parallel API available?

### DISCUSSION THREAD

**Discussion by jrwrigh** [CGNS/CGNS#790] [CGNStalk]
I've seen https://github.com/CGNS/CGNS/pull/715 was merged in quite a while ago, but the documentation for the API functions isn't up on the website. Are these functions stable? I'd be interested in using them and seeing what (if any) performance boost I can get.

**Comment by MicK7** [comment]
I think we need to release the new version to get the doc on the website. 
Hopefully, it's coming soon.
You can still test using the develop branch. I you find any problems with it, we could tackle them before release.
